test-at/pom.xml

```diff
@@ -11,6 +11,17 @@
 
     <artifactId>test-at</artifactId>
 
+    <repositories>
+        <repository>
+            <id>sonatype-oss-snapshot</id>
```

Comment:
I copy directly from an EmbeddedCassandra example. My opinion is: let this lines and wait to a new Release of the package.